### ISSUE TITLE
Organize Imports - How to use as3mxml.organizeImportsInDirectory?

### ISSUE THREAD

**Issue by ylazy** [BowlerHatLLC/vscode-as3mxml#678]
Hi!

I need to run Organize Imports with all files in my source paths. I found this command in the Keyboard Shortcuts panel:
![image](https://user-images.githubusercontent.com/6010183/229293309-acaad9b4-4a49-4772-bab5-a58d514babbd.png)
But I don't know how to use it. I can't search it in the command palette.

Thanks!

**Comment by joshtynjala** [comment]
Right-click the directory in VSCode's explorer. It should be called _Organize Imports (Recursive)_.

**Comment by joshtynjala** [comment]
It looks like you have a function that doesn't specify a return type, and I guess I forgot to check for null. It's an easy fix.
